internal/cli/install.go

```diff
@@ -117,6 +117,34 @@ var neededOptions = kubernetes.InstallationOptions{
 		Default:     "",
 		Value:       "",
 	},
+	{
+		Name:        "external-registry-url",
+		Description: "Provide a url for an external registry. This disables installation of the internal registry.",
+		Type:        kubernetes.StringType,
+		Default:     "",
+		Value:       "",
+	},
+	{
+		Name:        "external-registry-namespace",
+		Description: "Provide a namespace (or organization) of the external registry to which you have push access. It can be left empty.",
```

Comment:
```suggestion
		Description: "Provide namespace (or organization) of the external registry to which you have push access. It can be left empty.",
```